transform/snowflake-dbt/models/focalboard/focalboard_activity.sql

```diff
@@ -1,7 +1,7 @@
 {{config({
     "materialized": "incremental",
     "schema": "focalboard",
-    "tags":"hourly",
+    "tags":["nightly"],
```

Comment:
Downstream `hourly` models are using this model. It's only one (`focalboard_fact`), so I'd assume it can also be converted to nightly.

Reply:
I thought `focalboard_fact` is already on "nightly"? - https://github.com/mattermost/mattermost-data-warehouse/blob/cf612c08bd59682b67928f37a129579e891f630b/transform/snowflake-dbt/models/focalboard/focalboard_fact.sql#L9 ?